domains/jhw.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Jason Wohlgemuth's resume",
+    "repo": "https://github.com/jhwohlgemuth/resume",
+    "owner": {
+        "username": "jhwohlgemuth",
+        "twitter": "jhwohlgemuth"
+    },
+    "record": {
+        "CNAME": "jhwohlgemuth.github.io/resume"
```

CNAME can't contain `/`
```suggestion
        "CNAME": "jhwohlgemuth.github.io"
```